### DISCUSSION TITLE
Show the title only for the first element in version dropdown

### DISCUSSION THREAD

**Discussion by bugblasterX** [jimporter/mike#234] [Q&A] [answered]
Hi,
I'm trying to figure out how can i show only the title/alias of the first element in the version dropdown. For example I want to have "latest" only to be shown, not 1.1.2(latest). I thought of setting the --title "latest" during mike deploy, but in this case with a new deploy the previous version will show the title as well, while i want number.

**Comment by jimporter** [comment, marked as answer]
[`mike retitle`](https://github.com/jimporter/mike?tab=readme-ov-file#changing-a-versions-title) should do the trick.
